EIPS/eip-2537.md

```diff
@@ -277,10 +251,12 @@ MSMs are expected to be performed by Pippenger's algorithm (we can also say that
 
 To avoid non-integer arithmetic, the call cost is calculated as `(k * multiplication_cost * discount) / multiplier` where `multiplier = 1000`, `k` is a number of (scalar, point) pairs for the call, `multiplication_cost` is a corresponding single multiplication call cost for G1/G2.
```

Comment:
We remove MUL, so we have to update this reference: "multiplication_cost"

Reply:
I think the intent is that removing the precompile is separate from the gas schedule of a multiplication

my read is that `multiplication_cost` here would be either the G1 or G2 "base" gas cost depending on the nature of the MSM

Reply:
going to go ahead and approve, we can update this EIP later if needed, but it looks good to me at the moment